Description: Use Linux edition instead of jar
 Newer version of burpsuite might be built with newer version of Java
 hence Debian might not contains the correct version to run it.
 This patch switches download version from Jar to Linux, which
 contains jre runtime
 .
 burpsuite (2024.2.1.3-1parrot2) stable; urgency=medium
 .
   * Try install with Linux edition, fix JRE runtime problem
Author: Nong Hoang Tu <dmknght@parrotsec.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout https://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: (upstream|backport|vendor|other), (<patch-url>|commit:<commit-id>)
Bug: <upstream-bugtracker-url>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: (no|not-needed|<patch-forwarded-url>)
Applied-Upstream: <version>, (<commit-url>|commit:<commid-id>)
Reviewed-By: <name and email of someone who approved/reviewed the patch>
Last-Update: 2024-04-09

--- burpsuite-2024.2.1.3.orig/Makefile
+++ burpsuite-2024.2.1.3/Makefile
@@ -1,2 +1,4 @@
 install:
-	wget -q "https://portswigger-cdn.net/burp/releases/download?product=community&type=Jar" -O burpsuite.jar
+	wget -q "https://portswigger-cdn.net/burp/releases/download?product=community&type=Linux" -O burpsuite.sh
+	chmod +x burpsuite.sh
+	./burpsuite.sh -q -dir burpsuite
